app/_includes/header.html

```diff
@@ -29,9 +29,18 @@ <h1>{{ page.header_title }}</h1>
         {% endif %}
       </div>
 
-      {% if page.header_btn_text %}
+      {% if page.header_btn_text or page.header_btn_repo_href %}
         <div class="page-header-right">
-          <a class="page-header-btn" href="{{ page.header_btn_href | liquify }}" target="{{ page.header_btn_target }}">{{ page.header_btn_text | liquify }}</a>
+          {% if page.header_btn_repo_href %}
+            {% if header_btn_repo_text %}
+              <a class="page-header-btn" href="{{ page.header_btn_repo_href | liquify }}" target="{{ page.header_btn_repo_target }}">{{ page.header_btn_repo_text | liquify }}</a>
+            {% else %}
+              <a class="page-header-btn page-header-btn-repo" href="{{ page.header_btn_repo_href | liquify }}" target="{{ page.header_btn_repo_target }}">GitHub</a>
```

Comment:
Should this say "Source Code" instead of GitHub? "GitHub" seems vague (where on GitHub does this button link to?) and the icon already communicates that it will lead there anyway.

Reply:
I can change it to `Source Code`. I don't feel strongly on this.

Reply:
Let's do it then.

Reply:
Oh, forgot to reply, it is done.